beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/aws/S3Client.java

```diff
@@ -73,10 +73,21 @@ List<String> deleteObjects(String bucket, List<String> keys) {
       return Collections.emptyList();
     }
     if (!dryRunEnabled) {
-      keys.forEach(key -> log.info("Deleting: \"{}/{}\"", bucket, key));
-      DeleteObjectsRequest deleteObjectsRequest = new DeleteObjectsRequest(bucket)
-          .withKeys(keys.toArray(new String[] {}));
-      DeleteObjectsResult deleteObjectsResult = amazonS3.deleteObjects(deleteObjectsRequest);
+      log.info("Attempting to delete a total of {} objects, from [{}] to [{}]", keys.size(), keys.get(0), keys.get(keys.size() - 1));
```

Comment:
Logs are different. Is this expected?

Reply:
logs were having a negative effect on tracking issues, first, it stated it was deleting files before actually deleting them, which was misleading at the moment of tracking the error, it also happened to add too many logs that rendered previous logs to be lost on the pods, so i thought of making them shorter and more useful

Reply:
i agree, the logs may be more useful in this new format